test/eredis_sub_tests.erl

```diff
@@ -2,9 +2,6 @@
 
 -include_lib("eunit/include/eunit.hrl").
 -include("eredis.hrl").
--include("eredis_sub.hrl").
-
--import(eredis, [create_multibulk/1]).
```

Comment:
I decided to remove this, since it makes for more difficult to read code.

Reply:
👍